.devcontainer/docker-compose.yml

```diff
@@ -20,6 +19,8 @@ services:
     depends_on:
       - nats
     command: sleep infinity
+    ports:
+      - "3001:3001"
```

Comment:
This is a good idea, but I've run into issues where I try to run two compose projects and find out I have port collisions. For other services, I've made this available but commented it out, which I feel like is a nice compromise: https://github.com/infratographer/permissions-api/blob/b5f6e6c34c22ba511270802a2a0dacef36ea00ff/.devcontainer/docker-compose.yml#L21-L22

Maybe not necessary here though. YMMV

Reply:
I think... I would prefer this one to stay defaulted like this to be honest, just because I have run this devcontainer a lot in conjunction with the other projects I think somebody would be likely to have open at the same time (the governor UI, all the addons, permissions-api as of late etc) and those don't collide. The downside of defaulting it commented out like that would be requiring devs to catch one more setup thing in the readme and getting questions when they don't catch it... and we already get plenty of questions around correct env values for dev and such as-is.

tl;dr I think it's unlikely to be an issue here specifically and I'm afraid of adding more barriers to potential onboarding devs